cluster.go

```diff
@@ -51,7 +51,7 @@ type member struct {
 }
 
 // newCluster returns a new marathon cluster
-func newCluster(client *http.Client, marathonURL string) (*cluster, error) {
+func newCluster(client *http.Client, marathonURL string, DCOSPath string) (*cluster, error) {
```

Comment:
You are right, all the `api*()` behave similarly but `registerSSESubscription()` is [creating n URL](https://github.com/gambol99/go-marathon/blob/b307c574d90c358369555b8c694ac85cff4e76ac/subscription.go#L176) wich doesn't have **DCOSPath** into account.

What do you want me to do? Get rid of the Marathon path (`/marathon`) condition when **DCOSToken** is set?